src/main/java/org/jabref/logic/cleanup/DoiCleanup.java

```diff
@@ -14,23 +16,57 @@
 import org.jabref.model.entry.identifier.DOI;
 
 /**
- * Formats the DOI (e.g. removes http part) and also moves DOIs from note, url or ee field to the doi field.
+ * Formats
+ * the
+ * DOI
+ * (e.g.
+ * removes
+ * http
+ * part)
+ * and
+ * also
+ * moves
+ * DOIs
+ * from
+ * note,
+ * url
+ * or
+ * ee
+ * field
+ * to
+ * the
+ * doi
+ * field.
```

Comment:
Something seems to have gone wrong here 😃

Reply:
(apparently on most or all comments)

Reply:
The problem above is because of the mistake while configuring format file :relieved: . Thanks for pointing that and the format should be correct now.

Reply:
Sse here for the codestyle: https://github.com/JabRef/jabref/tree/main/config
The exceptions seem also to be wrapped unnecessarily